src/pages/dev/general-message-passing/solidity-utilities.mdx

```diff
@@ -1,33 +1,55 @@
 # Solidity utilities
 
-To facilitate cross-chain development, we have provided some Solidity utilites. These can be found [here](https://github.com/axelarnetwork/axelar-utils-solidity). Each of these is described below, along with their proposed use.
+To facilitate interchain development, we have provided some Solidity utilites. These can be found [here](https://github.com/axelarnetwork/axelar-utils-solidity). Each of these is described below, along with their proposed use.
```

Comment:
should link to gmp-sdk here instead of the old github url. Also, create3 needs gmp-sdk v5.3+